code/api/documents/views.py

```diff
@@ -6,8 +7,34 @@ def index(request):
 
     query_params = request.GET
 
-    search_engine = engine.Engine()
+    # TODO: Add bounds checking on inputs
 
-    docs = search_engine.query_corpus(query_params.get("search", ""), 5)
+    # "search" is the only mandatory parameter
+    # Set defaults for any unset parameters
+    if query_params.get("corpus", "") == "" or query_params.get("corpus", "") == None:
+        corpus = "lectures"
+    else:
+        corpus = query_params.get("corpus", "")
```

Comment:
Would it be possible to shorten this to something like this?

```python
corpus = query_params.get('corpus', 'lectures') or 'lectures'
```

I ran a few tests in the python console:

```python
>>> hash = {'test_empty_string': '', 'test_none': None}
>>> hash.get('test_empty_string')
''
>>> hash.get('test_empty_string', 'default')
''
>>> hash.get('test_empty_string', 'default') or 'default'
'default'
>>> hash.get('test_none')
>>> hash.get('test_none', 'default')
>>> hash.get('test_none', 'default') or 'default'
'default'
>>> hash.get('test_not_exist')
>>> hash.get('test_not_exist', 'default')
'default'
>>> hash.get('test_not_exist', 'default') or 'default'
'default'
```

Reply:
Absolutely, that's a much better approach.